src/HttpRequest.cc

```diff
@@ -93,7 +93,6 @@ HttpRequest::init()
     error.clear();
     peer_login = nullptr;      // not allocated/deallocated by this class
     peer_domain = nullptr;     // not allocated/deallocated by this class
-    peer_host = nullptr;
```

Comment:
BTW, should not we also remove other similar fields, i.e., peer_domain and peer_login?

Reply:
Ok.